blueocean-dashboard/src/main/js/util/logDisplayHelper.js

```diff
@@ -26,12 +26,28 @@ export const getNodesInformation = (nodes) => {
         .filter((item) => item.state === null && item.result === null).length !== nodes.length;
   // principal model mapper
     let wasFocused = false; // we only want one node to be focused if any
+    let parallelNodes = [];
+    let parent;
     const model = nodes.map((item, index) => {
         const hasFailingNode = item.edges ? item.edges
           .filter((itemError) => errorNodes.indexOf(itemError.id) > -1).length > 0 : false;
         const isFailingNode = errorNodes.indexOf(item.id) > -1;
         const isRunningNode = runningNodes.indexOf(item.id) > -1;
 
+        /*
+         * are we in a node that indicates that we have parallel nodes?
+         */
+        if (item.edges && item.edges.length >= 2) {
+            parallelNodes = item.edges.map((itemParallel) => itemParallel.id);
+        }
+        // in case we had been in a parallel node before, we will indicate it and remove the id of the parallel array
+        const indexParallel = parallelNodes.indexOf(item.id);
+        const isParallel = indexParallel > -1;
```

Comment:
NIT: Maybe evaluate in a way that's consistent with the wording of all the docs on `indexOf`. Every doc on this function that I have ever seen says "not equal to -1" (or very similar). From that, I'd always find `const isParallel = indexParallel !== -1;` to be more consistent. `> -1` is obviously not wrong though, so just a NIT 😄